docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+# This base config defines the production containers.
+
+version: "3"
+
+services:
+  # Node-based static asset build tooling
+  assets:
+    build: ./assets
+    image: wdfn_assets
+    ports:
+      - "9000:9000"
+  # Node server-renderer
+  graph_server:
+    privileged: true
```

Comment:
Why is this necessary?

Reply:
I suspect this may be a deployment issue and in general a problem, but Chrome won't run without it. There's follow-up work needed to determine if there is a viable alternative.